db/migrate/20240422150502_require_address1_and_city_and_phone_and_country_and_on_address.rb

```diff
@@ -0,0 +1,8 @@
+class RequireAddress1AndCityAndPhoneAndCountryAndOnAddress < ActiveRecord::Migration[7.0]
+  def change
+    change_column_null :spree_addresses, :address1, false
+    change_column_null :spree_addresses, :city, false
+    change_column_null :spree_addresses, :phone, false
+    change_column_null :spree_addresses, :country_id, false
```

Comment:
Unfortunately, we got a problem here, too:
```
csicsoka.org | CHANGED | rc=0 >>
 count 
-------
    21
```

This one should be quite easy to fix by setting the default country on this instance, manually in the database.

Reply:
It would also be good to check if these invalid addresses are actually used. Unfortunately, we have 13 references from other tables to addresses. We should probably script that.